Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM adzerk/boot-clj:latest as builder
+
+ENV BOOT_VERSION=2.7.2
+ENV BOOT_CLOJURE_VERSION=1.8.0
+
+WORKDIR /usr/local/github-changelog
+
+COPY . .
+RUN cd /usr/local/github-changelog && boot uberjar
```

Comment:
I just realized that the `WORKDIR` is already `/usr/local/github-changelog`, so I believe the `cd` could be avoided. Also we should combine the two `RUN` commands into one.

Reply:
Let me try this.

Reply:
As I could see `WORKDIR` doesn't have any effect when you run command with `RUN`.
Probably the merging could be done.

Reply:
It seems to be working for me.